frontend/src/pages/OnboardingPageV2/OnboardingPageV2.styles.scss

```diff
@@ -0,0 +1,233 @@
+.setup-flow {
+	margin-top: 74px;
+	z-index: 1;
+
+	&__header {
+		background: rgba(11, 12, 14, 0.7);
```

Comment:
use variable from the design tokens

Reply:
There are few changes need to made post this feature is done with the functionality. I will fix this once the flow is functional.